### DISCUSSION TITLE
how was this struct located?

### DISCUSSION THREAD

**Discussion by nekwo** [Sixze/ALS-Refactored#425] [Q&A]
![image](https://github.com/Sixze/ALS-Refactored/assets/43717185/082ed1cc-9838-4392-b25c-d975413795fe)

how was this struct located in the editor? i copied and pasted the variable but i'm very curious it's not in the dropdown when i search for it

**Comment by jomath229** [comment]
Problem in 5.3 where structs that have uint8 won't show up. There's a new thread about it. Change the uint8 to bools in the struct and it will show up properly.
